addons/html_editor/static/src/main/link/link_popover.xml

```diff
@@ -85,10 +86,18 @@
                             </div>
                         </t>
                         <t t-if="props.allowTargetBlank">
-                            <div class="d-flex mb-1 target-blank-option">
-                                <input class="me-1" type="checkbox" t-att-checked="state.linkTarget === '_blank'" t-on-change="(ev)=>this.state.linkTarget = ev.target.checked ? '_blank' : ''"/>
-                                <label>Open in new window</label>
-                            </div>
+                            <t t-if="!state.isDocument || (state.isDocument and !state.directDownload)">
+                                <div class="d-flex mb-1 target-blank-option">
+                                    <input class="me-1" type="checkbox" t-att-checked="state.linkTarget === '_blank'" t-on-change="(ev)=>this.state.linkTarget = ev.target.checked ? '_blank' : ''"/>
+                                    <label>Open in new window</label>
+                                </div>
+                            </t>
+                            <t t-if="state.isDocument">
+                                <div class="d-flex mb-1">
+                                    <input class="me-1 direct-download-option" type="checkbox" t-model="state.directDownload" t-on-change="onClickDirectDownload"/>
+                                    <label>Direct download</label>
+                                </div>
+                            </t>
```

Comment:
Reoder:
first checkbox: Direct download
second checkbox: Open in new window

Why not use Checkbox component ?